cli/pcluster/config_sanity.py

```diff
@@ -343,6 +343,11 @@ def validate(self, resource_type, resource_value):  # noqa: C901 FIXME
             ):
                 self.__fail(resource_type, "DNS Hostnames not enabled in %s" % resource_value)
         # VPC Subnet Id
+        elif resource_type == "VPC_COHERENCE":
+            cidr_value = resource_value[0]
+            public_ip = resource_value[1]
+            if cidr_value and public_ip is False:
+                self.__fail(resource_type, "compute_subnet_cidr needs use_public_ips to be true")
```

Comment:
why is this required? is it always true? I'm wondering if it's a bug or a necessary requirement